apis/train.py

```diff
@@ -1,8 +1,76 @@
+import os
+import pickle
+import pandas as pd
+
+from numpy.random import RandomState
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import roc_auc_score
+from datetime import datetime
 from flask_restplus import Namespace, Resource, fields
+from apis.config import MODELS_ROOT, TEMP_CSV
 
 api = Namespace('train', description='Namespace for training')
 
 @api.route('/fit')
 class Train(Resource):
+
     def get(self):
-        return {'success': 'Training pipeline successful'}
\ No newline at end of file
+        return self.train() # {'success': 'Training pipeline successful',}
+
+    def train(self, file=None, target_var=None):
+
+        # curretnly it's having nothing as i/p
+        start_time = datetime.now()
+
+        if target_var is None:
+            response = "Outcome"
+        else:
+            response = target_var
+
+        if file is None: # sample file
+            df = pd.read_csv(TEMP_CSV)
+        else:
+            df = pd.read_csv(file)
+
+        train, valid = self._split_dataset(df, 0.20, 12345)
+        predictors   = df.columns[:-1]
```

Comment:
Use code-reformatter. Or pretty fiery from VSCode. Uneven spacing :)

Reply:
this was on purpose actually;

Reply:
Linting will fail 😄